pyiode/iode_database/abstract_database.pyx

```diff
@@ -415,6 +417,8 @@ cdef class _AbstractDatabase:
         cdef CKDBAbstract* other_db_ptr = (<_AbstractDatabase>other).abstract_db_ptr
         self.abstract_db_ptr.merge(dereference(other_db_ptr), <bint>overwrite)
 
+    # GDM> I find the function name confusing, with that name and signature, I expected the opposite flow of data
+    #      I think copy_from would be a better name
```

Comment:
Well, in the old GUI, this fonction is called from the item menu `Copy Into Work Space`. So I simply choose `copy_into` as a shortcut for `copy_into_workspace`. But reading your comment make me think it would be better (and less misleading) to use `copy_from` instead. In a Python script, `copy_from` would make more sense.